Gruntfile.js

```diff
@@ -204,11 +204,20 @@ module.exports = function(grunt) {
           name: 'ungit',
           dir: './build/resource',
           out: './build',
+          icon: './icon.ico',
```

Comment:
electron-packager does not accept .png format as application icon unfortunately.